lib/resources/r4/appointment.yaml

```diff
@@ -25,6 +25,23 @@ fields:
     }
   note: When creating a new appointment, the status must be `booked` or `proposed`.
 
+- name: serviceCategory
+  required: 'No'
+  type: CodeableConcept
+  description: A broad categorization of the service that is to be performed during this appointment.
+  binding:
+    description:  Additional details about the categorization of the service.
+    terminology:
+    - display: SNOMED CT
+      system: http://snomed.info/sct
+      info_link: https://hl7.org/fhir/R4/snomedct.html
+    - display: appointmentReason
+      system: http://terminology.hl7.org/CodeSystem/v2-0276
+      info_link: https://terminology.hl7.org/2.1.0/CodeSystem-v2-0276.html
+    - display: Millennium Scheduling Process Options
+      system: https://fhir.cerner.com/&lt;EHR source id&gt;/codeSet/16127
+      info_link: https://fhir.cerner.com/millennium/r4/proprietary-codes-and-systems/#code-set-16127-scheduling-process-options
```

Comment:
should this be 'code-set-16127-scheduling-processing-options'? wonder if this link works! Cqn we double check please.

Reply:
Good catch https://github.com/cerner/fhir.cerner.com/pull/585/commits/eda9cc99bab0fe6f02a2c16f0400e236b74de0f6